app/javascript/components/StudyResults.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react'
+import Tab from 'react-bootstrap/lib/Tab'
+import Tabs from 'react-bootstrap/lib/Tabs'
+import { useTable, usePagination } from 'react-table'
+
+/**
+ * Wrapper component for studies tab on homepage
+ */
+const StudyResults = props => {
+  return (
+    <Tab.Container id="result-tabs" defaultActiveKey="study">
+      <Tabs defaultActiveKey='study' animation={false} >
+        <Tab eventKey='study' title="Studies" >
+          <StudiesTabContent turnPage ={props.handlePageTurn} studies= {props.results}/>
+        </Tab>
+        <Tab eventKey='files' title="Files"></Tab>
+      </Tabs>
+    </Tab.Container>
+  )
+}
+
+/**
+ * Component for the content of the 'Studies' tab
+ */
+const StudiesTabContent = ({ studies, turnPage }) => {
+  const columns = React.useMemo(
+    () => [{
+      accessor: 'study',
+    }])
+
+  const data = studies.studies.map(result => (
+    {
+      study: <Study
+        study={result}
+        key={result.accession}
+        className='card'
+      />,
+    }
+  ),
+  )
+
+  const {
+    getTableProps,
+    getTableBodyProps,
+    prepareRow,
+    rows,
+    canPreviousPage,
+    canNextPage,
+    pageCount,
+    gotoPage,
+    nextPage,
+    previousPage,
+    state: { pageIndex, pageSize },
+  } = useTable({
+    columns,
+    data,
+    // holds pagination states
+    initialState: {
+      pageIndex: studies.currentPage -1,
+      // This will change when there's a way to determine amount of results per page via API endpoint
+      pageSize: 5,
+    },
+    pageCount: studies.totalPages,
+    manualPagination: true,
+  },
+  usePagination)
+  return (
+
+    <Tab.Content id ='results-content'>
```

Comment:
I would add a comment here directly linking to the react-table example you based this off of, since this wasn't easy to figure out!

Reply:
eacadd37